### ISSUE TITLE
npm package is out of date

### ISSUE THREAD

**Issue by mklaber** [fullscale/elastic.js#90]
Not sure why, but the dist file that npm spits out different than the one in `master`. Specifically, it's missing https://github.com/fullscale/elastic.js/blob/master/dist/elastic.js#L15769-L15772

This should fail to find anything via `grep`:

``` bash
curl -O http://registry.npmjs.org/elastic.js/-/elastic.js-1.2.0.tgz
tar -zxvf elastic.js-1.2.0.tgz
grep 'fast-vector-highlighter' package/dist/elastic.js
```

(registry.npmjs.org URL comes from the `dist.tarball` property in the `package.json` file `npm install elastic.js` grabs)

**Comment by timcroydon** [comment]
Yes, I found a few missing features introduced after the last release version. At present, as a workaround, I'm using the following line in my `package.json` to load the latest code, using npm's native GitHub support (see https://docs.npmjs.com/files/package.json#github-urls):

```
"elastic.js": "fullscale/elastic.js#b29b7d2"
```

**Comment by dmitryt** [comment]
I want to mention here, that version `1.1.1` doesn't contain all files from `dist` folder.
That's because `files` in package.json looks like this:
```
"files": [
    "elastic-node-client.js",
    "elastic.js"
  ],
```

**Comment by sudo-suhas** [comment]
@dmitryt, @mklaber, @timcroydon This library hasn't been updated in years. You could try `elastic-builder` if you are using `elasticsearch@2` or higher. You can also try it out in the online playground - https://elastic-builder.js.org/.
